shacl12-node-expr/index.html

```diff
@@ -1583,7 +1592,288 @@ <h3>OrderBy Expressions</h3>
+								<div class="turtle">
+ex:CompanyShape
+    a sh:NodeShape ;
+    sh:targetClass ex:Company ;
+    sh:property ex:CompanyShape-allSkills .
+
+ex:CompanyShape-allSkills
+    a sh:PropertyShape ;
+    sh:path ex:allSkills ;
+    sh:name "all skills" ;
+    sh:values [
+		shnex:nodes [
+			shnex:pathValues ex:employee ;
+		] ;
+		shnex:flatMap [
+			shnex:pathValues ex:skill ;
+		] ;
+    ] .
+								</div>
+								<div class="jsonld">
+									<equivalent jsonld>
+								</div>
+							</div>
+						</aside>
+					</p>
+				</section>
+
+				<section id="FindFirstExpression">
+					<h3>FindFirst Expressions</h3>
+					<p class="syntax">
+						<span data-syntax-rule="FindFirstExpression-syntax">
+							A <a>blank node</a> that is the <a>subject</a> of the following properties
+							is called a <dfn>findFirst expression</dfn> with the <a>function name</a> <code>shnex:FindFirstExpression</code>:
+							<table class="term-table">
+								<thead>
+									<th>Property</th>
+									<th>Constraints</th>
+									<th>Description</th>
+								</thead>
+								<tbody>
+									<tr>
+										<td><b><code>shnex:findFirst</code></b></td>
+										<td>
+											A <a>well-formed</a> <a>shape</a>.
+										</td>
+										<td>
+											The shape that the matching node must conform to.
+										</td>
+									</tr>
+									<tr>
+										<td><b><code>shnex:nodes</code></b></td>
+										<td>
+											A <a>well-formed</a> <a>node expression</a>.
+										</td>
+										<td>
+											The input nodes. If omitted, defaults to the focus node.
+										</td>
+									</tr>
+								</tbody>
+							</table>
+						</span>
+					</p>
+					<div class="def" id="FindFirstExpression-evaluation">
+						<div class="def-header">EVALUATION OF FINDFIRST EXPRESSIONS</div>
+						<p>
+							Let <code>shape</code> be the <a>value</a> of <code>shnex:findFirst</code>
+							and <code>nodes</code> be the <a>value</a> of <code>shnex:nodes</code> in a <a>findFirst expression</a>.
+							If <code>shnex:nodes</code> is not specified, let <code>nodes</code> be the focus node.
+							Let <code>N</code> be the <a>output nodes</a> of <code>evalExpr(nodes, focusGraph, focusNode, scope)</code>.
+							The <a>output nodes</a> of the <a>findFirst expression</a> contain exactly the first node <code>n</code> 
+							in <code>N</code> that <a>conforms</a> to the <a>shape</a> <code>shape</code>, 
+							or an empty sequence if no such node exists.
+						</p>
+					</div>
+					<p><em>The remainder of this section is informative.</em></p>
+					<p>
+						The <code>shnex:findFirst</code> operation finds the first node in a sequence that conforms to a given shape.
+					</p>
+					<p id="FindFirstExpressionExample">
+						The following example illustrates the use of <code>shnex:findFirst</code> to derive a property
+						<code>ex:seniorEmployee</code> that finds the first employee with more than 5 years of experience.
```

Comment:
Generally, in streaming prose, it's better to use words instead of numerals for numbers lower than ten or twenty.
```suggestion
						<code>ex:seniorEmployee</code> that finds the first employee with more than five years of experience.
```